src/templates/main.html

```diff
@@ -1,7 +1,120 @@
 <main>
-    <img class="decoration" src="./images/decoration.png" alt="decoration">
-    <h2>Hello Awesome App!</h2>
-    <p>To get started edit a file in "./src" and save to reload.</p>
-    <button class="button-cta">Get Started!</button>
-    <button class="button">Find out more</button>
+    <div class="flex-container">
+        <div class="flex-column to-do" id="idTODO">
+            <div class="header">
+                <div class="name-column">TO DO</div>
+                <div class="edit-column"><i class="fas fa-ellipsis-h"></i>
+                    <div class="popup-dots">
+                        <div class="edit-name-column">Edit column name</div>
+                    </div>
+                </div> 
+                <div class="add-column"><i class="fas fa-plus"></i></div>
+            </div>
+            <div class="content-column" id="contentTODO"></div>
+        </div>
+        <div class="flex-column">
+            <div class="header">
+                <div class="name-column">IN PROGRESS</div>
+                <div class="edit-column"><i class="fas fa-ellipsis-h"></i>
+                    <div class="popup-dots">
+                        <div class="edit-name-column">Edit column name</div>
+                        <div class="delete-column">Delete column</div>
+                    </div>
+                </div> 
+                <div class="add-column"><i class="fas fa-plus"></i></div>
+            </div>    
+            <div class="content-column"></div>
+        </div>
+        <div class="flex-column">
+            <div class="header">
+                <div class="name-column">REVIEW</div>
+                <div class="edit-column"><i class="fas fa-ellipsis-h"></i>
+                    <div class="popup-dots">
+                        <div class="edit-name-column">Edit column name</div>
+                        <div class="delete-column">Delete column</div>
+                    </div>
+                </div>
+                <div class="add-column"><i class="fas fa-plus"></i></div>
+            </div>
+            <div class="content-column"></div>
+        </div> 
+        <div class="flex-column">
+            <div class="header">
+                <div class="name-column">IN TESTING</div>
+                <div class="edit-column"><i class="fas fa-ellipsis-h"></i>
+                    <div class="popup-dots">
+                        <div class="edit-name-column">Edit column name</div>
+                        <div class="delete-column">Delete column</div>
+                    </div>
+                </div>
+                <div class="add-column"><i class="fas fa-plus"></i></div>
+            </div>    
+            <div class="content-column"></div>
+        </div>
+        <div class="flex-column">
+            <div class="header">
+                <div class="name-column">ACCEPTANCE PO</div>
```

Comment:
Do you use this html >?

Reply:
nu inteleg prea bine intrebarea... la care element te referi mai exact?

Reply:
`<div class="name-column">ACCEPTANCE PO</div> `

Mai precis, daca e folosit atunci da de inteles ca nu se poate add-ul de coloana.  Si parca il vazusem la meeting.